build.gradle

```
@@ -1074,8 +1074,12 @@ publishing {
 
 signing {
     sign publishing.publications.mavenJava
+    if (!findProperty("signing.keyId") || findProperty("signing.keyId").equals("")) {
```

```suggestion
    if (!findProperty("signing.keyId")) {
```

In Groovy, both null and the empty string are falsy.